dataactcore/scripts/fix_split_error_metadata.py

```diff
@@ -0,0 +1,78 @@
+import logging
+
+from dataactcore.interfaces.db import GlobalDB
+from dataactcore.logging import configure_logging
+
+from dataactvalidator.health_check import create_app
+
+logger = logging.getLogger(__name__)
+
+
+def duplicate_removal(table):
+    """ Run the sql to remove duplicates
+
+        Args:
+            table: The table to remove the duplicates from
+    """
+    logger.info('Fixing {} duplicated entries'.format(table))
+
+    # Creating a temporary table storing all the duplicates
+    create_table_sql = """
+                CREATE TABLE IF NOT EXISTS temp_duplicated_{} (
+                    job_id integer,
+                    original_rule_label text,
+                    occurrences integer,
+                    min_id integer
+                );
+            """
+    sess.execute(create_table_sql.format(table))
+    # In case something went wrong, we don't want extra data
+    sess.execute('TRUNCATE TABLE temp_duplicated_{};'.format(table))
+
+    # Inserting a summary of all duplicated error metadata
+    duplicates_insert_sql = """
+                INSERT INTO temp_duplicated_{table} (job_id, original_rule_label, occurrences, min_id)
```

Comment:
Looks like this query is over-indented, at least compared to the other queries in this script.